<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="warehouseWiew"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="buttonBack">
          <property name="text">
           <string>&lt;&lt;&lt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonHome">
          <property name="text">
           <string>Home</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonNext">
          <property name="text">
           <string>&gt;&gt;&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="openFileButton">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Open File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Run CGP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addFileButton">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Add File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cgpButton">
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>Show Setup</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionImport_warehouse"/>
    <addaction name="actionCreate_warehouse"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_file"/>
    <addaction name="actionCreate_file"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionCGP_settings"/>
    <addaction name="actionEditor_settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUser_manual"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionImport_warehouse">
   <property name="text">
    <string>Import warehouse</string>
   </property>
  </action>
  <action name="actionCreate_warehouse">
   <property name="text">
    <string>Create warehouse</string>
   </property>
  </action>
  <action name="actionAdd_file">
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionCreate_file">
   <property name="text">
    <string>Create file</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionCGP_settings">
   <property name="text">
    <string>CGP settings</string>
   </property>
  </action>
  <action name="actionEditor_settings">
   <property name="text">
    <string>Editor settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUser_manual">
   <property name="text">
    <string>User manual</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
